.gitignore

```
@@ -9,37 +9,17 @@ x64
 *.bin
 wpop_*.txt
 out/
-build
```

This is a useful directory...

---

I suppose this was a issue mentioned above? Can you explain more?

---

It isn't. Official `build.md` doc also suggest using `build/`.

---

That indicates to create a new build directory. But not include it in source code repository, neither CMakeLists added build subdirectory.

---

Not really: https://github.com/mrc-ide/covid-sim/blob/master/docs/build.md#building-with-makefiles

---

Yes you should have build directory on your own behalf which is a CMake convention.

---

When you clone the repository, cd to it and execute the command, you'll have `build/` directory inside the git repository. This PR would make a lot of unwanted files to be tracked by Git. I still don't understand the need to remove this line.

---

I think what is being missed is the line should be:

```suggestion
build/
```

That is with a trailing `/` to indicate it is a directory.

---

I see. I had this change locate under vscode section. I have changed it back now. Hope it would not mislead to the disscuss above this time.